examples/task-tracker/client/tasks/update_task_parameters.go

```diff
@@ -49,19 +49,30 @@ func NewUpdateTaskParamsWithContext(ctx context.Context) *UpdateTaskParams {
 	}
 }
 
+// NewUpdateTaskParamsWithHTTPClient creates a new UpdateTaskParams object
+// with the default values initialized, and the ability to set a custom HTTPClient for a request
+func NewUpdateTaskParamsWithHTTPClient(client *http.Client) *UpdateTaskParams {
+	var ()
+	return &UpdateTaskParams{
+		HTTPClient: client,
+	}
+}
+
 /*UpdateTaskParams contains all the parameters to send to the API endpoint
 for the update task operation typically these are written to a http.Request
 */
 type UpdateTaskParams struct {
 
 	/*Body
 	  The task to update
-
+	  Required: true
+	  In: body
 	*/
 	Body *models.Task
 	/*ID
 	  The id of the item
-
+	  Required: true
+	  In: path
 	*/
 	ID int64
```

Comment:
This ID parameter is required, but for now Required isn't validated in my added Validate Client Parameter code. Any idea how I would be able to add validation for required parameters @casualjim ?